content/en/docs/overview/singleton-sdk/Angular Client Introduction/translate-pipe.md

```diff
@@ -21,44 +21,66 @@ Get translation by key, will find the source in the sourceBundle according to th
 
 ```
 
-##### **L10n pipe** (deprecated)
-
-```
-{{ key | translate: [ : source [ : variables ] ] }}
+#### **Input**
 
-```
+| Parameter |  Type   | Required | <div style="text-align:center">Description</div>                                                                                                       |
+|:---------:|:-------:|----------|:-------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    key    | string  | required | Define the key to identify the translation, it should name like this: component_module_page_control_shortmsg. e.g. web_settings_stats_statsTable_host; |
+| variables | Array[] | optional | Parameter array, it's used to format the translation/source when they contain placeholders.                                                            |                                        |
 
-#### **Input**
 
-| Parameter |  Type  |                         Description                          |
-| :-------: | :----: | :----------------------------------------------------------: |
-|    key    | string | Define the key to identify the translation, it should name like this: component_module_page_control_shortmsg. e.g. web_settings_stats_statsTable_host; |
+#### **Example Code**
 
-#### **Parameters**
+```
+export const ENGLISH = {
+    'some.unique.key' : 'English for some.unique.key',
+    'VIP.description' : '{0} is common lib developed by VMware G11n team.',
+    'innerHTML.usage' : 'For more details, please refer to <a href=&quot;https://ngx.eng.vmware.com/@vmw/ngx-vip/translate-pipe/documentation&quot;>ngx-vip guide</a>.'
+    // innerHTML usage: use the innerHTML attribute with the pipe on any element. Not only for plain text node, also for raw HTML tags in source strings as inline tags.
+    // Note: Convert " to &quot; for the value of attribute.
+}
 
-| Parameter |  Type   | Required |                         Description                          |
-| :-------: | :-----: | :------: | :----------------------------------------------------------: |
-|  source   | String  |    No    | English string as default value, API will return it when there's no translation found either from cache or remote singleton server. |
-| variables | Array[] |    No    | Parameter array, it's used to format the translation/source when they contain placeholders. |
+```
 
 
-#### **Example Code**
+```
+// L10n pipe
+{{'some.unique.key' | vtranslate }}
 
-![translate-pipe-1](https://github.com/zmengjiao/singleton/raw/website/content/en/images/translate-pipe/translate-pipe-1.png)
+{{'VIP.description' | vtranslate: 'VIP for Angular client'}}
```

Comment:
{{'singleton.description' | vtranslate: 'Singleton for Angular client'}}

Reply:
Done!